ui/app/templates/components/transform-show-transformation.hbs

```diff
@@ -18,8 +18,8 @@
       </span>
     </div>
     <div class="copy-text level">
-      {{#let "vault write transform/encode/payments value=<enter your value here>" as |copyEncodeCommand|}}
-        <code>vault write transform/encode/payments value=&lt;enter your value here&gt;</code>
+      {{#let "vault write transform/encode/<your role name> value=<enter your value here> tweak=<base-64-string>" as |copyEncodeCommand|}}
```

Comment:
should also replace `transform` with `backend`

Reply:
Ah, good point!